tests/suites/helpers.function

```diff
@@ -8,16 +8,13 @@
 #include "mbedtls/platform.h"
 #else
 #include <stdio.h>
-#define mbedtls_printf     printf
```

Comment:
The concept and intent - if it wasn't clear - was to change all output to `fprinf()` to make it easier to redirect, hence `printf()` was removed as that can't be redirected without fuss. 

That's why it was removed.

Maybe we need a comment to say as much.